package/wireguard/package

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(wireguard)
+pkgdesc="Fast, modern, secure VPN tunnel"
+url=https://www.wireguard.com
+_wireguardver=1.0.20201221
+pkgver=$_wireguardver-1
+timestamp=2020-12-21T11:54Z
+section=utils
+maintainer="Jonah Weissman <jonahrweissman+toltec@gmail.com>"
+license=GPL-2.0
+depends=(wireguard-tools)
+
+_kernelrepo=https://github.com/remarkable/linux
+_kernelrevs=(
+    1774e2a6a091fdc081324e966d3db0aa9df75c0b
+    d4e7e07a390f8b2544ca09d69142d18114149004
+)
+_defconfigs=(
+    arch/arm/configs/zero-gravitas_defconfig
+    arch/arm/configs/zero-sugar_defconfig
+)
+_moddirs=(
+    4.9.84-zero-gravitas
+    4.14.78
+)
+
+image=base:v1.2.1
+source=(https://git.zx2c4.com/wireguard-linux-compat/snapshot/wireguard-linux-compat-"$_wireguardver".zip)
+sha256sums=(c74cedb10c9b830d937b5575e4b7d2625a8534d2c06a80368e5530db23e69d3b)
+
+build() {
+    apt-get update
+    apt-get install -y bc lzop git
+    git init
+    git remote add -f origin "$_kernelrepo"
```

Comment:
Probably better to mimic what was done here: https://github.com/toltec-dev/toltec/blob/a3db67b0c74e0d99edbdeb5fdb06aea45b21ae87/package/fbink/package#L18

Reply:
That's a good idea to do `--depth=1`, thanks!

Is it okay to track a branch rather than a specific revision? I was worried that it might make the build less reproducible.

The reason I didn't just do a git clone was that I wanted the files to appear directly in `/src`, so that I wouldn't have to make as many modifications to my existing recipe, but I realize now that I could do `git clone URL $(pwd)`.

Reply:
I'd suggest using a specific commit to ensure that the build is reproducible.

Reply:
Is this better?
https://github.com/toltec-dev/toltec/blob/27e2c318250d535b7d1de2644ce4f297405321dc/package/wireguard/package#L43-L44

It reduces the size of `.git/` from 1.9GB to 350MB

Reply:
Looks good to me!